lib/compilers/perl.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2019, Sebastian Rath
```

Comment:
Are you Sebastian Rath? Or is this from him? Do we have his OK for this? Just checking here - I'm a bit nervous about someone checking in code copyright to other people. It's also 2025 now. If this code was written in 2019 by Sebastian Rath and he's OK with this (which may be the case!) then we're ok :)

Reply:
Considering the base code is 99% with any other compiler code  and might have just come from that, I would say it would be reasonable to just change it to Compiler Explorer Authors